### PR TITLE
PFM 2177 and PFM 2174

### PR DESCRIPTION
**Pull request by hariomsinha-eGov**
This PR has changes for PFM 2174 ( Projects Create static screen ) and PFM 2177 (Project Details View static screen).

### CODE REVIEW

File: frontend/micro-ui/web/micro-ui-internals/packages/css/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@egovernments/digit-ui-works-css",
-  "version": "0.0.9",
+  "version": "0.0.10",
```
Comment (nipunarora-eGov) [inline frontend/micro-ui/web/micro-ui-internals/packages/css/package.json:3]: @hariomsinha-eGov update the version to .11

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/Project/src/pages/employee/ProjectDetails/ProjectDetailsNavDetails.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+    // integrate once api is ready
+    const workflowDetails = {}; 
+    const { applicationDetails_ProjectDetails, applicationData_ProjectDetails } = Digit.Hooks.project.useViewProjectDetails({}); 
+    const { applicationDetails_FinancialDetails, applicationData_FinancialDetails } = Digit.Hooks.project.useViewFinancialDetails({}); 
+
+    return (
+        <>
+        {
+            (activeLink === "Project_Details") && (
```
Comment (nipunarora-eGov) [inline frontend/micro-ui/web/micro-ui-internals/packages/modules/Project/src/pages/employee/ProjectDetails/ProjectDetailsNavDetails.js:14]: @hariomsinha-eGov For Project Details Card pls use this component -> \micro-ui-internals\packages\modules\Estimate\src\pageComponents\ViewProject.js

We'll make this a generic component later so whenever we need to see the project details card we'll pass the projectId as prop and make a search call

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/Project/src/pages/employee/ProjectDetails/ProjectDetailsNavDetails.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+                    showTimeLine={false}
+                    timelineStatusPrefix={""}
+                    businessService={""}
+                    forcedActionPrefix={"PROJECT"}
+              />
+            )
+        }
+        {
+            (activeLink === "Financial_Details") && (
+                <ApplicationDetails
```
Comment (nipunarora-eGov) [inline frontend/micro-ui/web/micro-ui-internals/packages/modules/Project/src/pages/employee/ProjectDetails/ProjectDetailsNavDetails.js:31]: Similarly make this financial details into a pageComponent for generic purposes @hariomsinha-eGov

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/Project/src/pages/employee/ProjectDetails/ProjectDetailsNavDetails.js (hunk 2 above)
Comment (hariomsinha-eGov) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/Project/src/pages/employee/ProjectDetails/ProjectDetailsNavDetails.js:15]: updated

**Review by nipunarora-eGov — APPROVED** [review]

**Review by jagankumar-egov — APPROVED** [review]

**Pull request merged**